client/index.ts

```diff
@@ -3,10 +3,6 @@ import {startApp} from 'superdesk-core/scripts/index';
 setTimeout(() => {
     startApp(
         [
-            {
-                id: 'broadcasting',
-                load: () => import('superdesk-core/scripts/extensions/broadcasting'),
```

Comment:
is it broken?

Reply:
it's duplicated, the same thing is imported on line 23, except with custom configuration so I chose to keep that